doc/content/getting-started/migrating/device-json.md

```diff
@@ -82,6 +84,10 @@ The linked specification is quite extensive, and contains a lot of fields that a
 ```
 {{</ note >}}
 
+## Migrating Devices with Existing Sessions
+
+Migrating devices with existing sessions is not currently supported in {{% tts %}}. Follow this [pull request](https://github.com/TheThingsNetwork/lorawan-stack/pull/3680) for progress on this feature.
```

Comment:
I don't think we should refer to the PR here, because even if it gets merged, it's not released or deployed yet.

Can we use the `new-in-version` label here for 3.11.3?

Or a new `coming-in-version` label?

Reply:
We could, but it seems strange. Proposed an update

Reply:
OK this is also fine with me